src/yaml/yaml.cr

```diff
@@ -1,5 +1,38 @@
 require "./*"
 
+# The YAML module serves to deserialize data from YAML into accessible structure.
```

Comment:
How about "The YAML module **deserializes YAML data** into **an** accessible structure" or similar?